debian/changelog

```diff
@@ -1,10 +1,11 @@
-cloud-init (23.3.2-0ubuntu0~20.04.1) UNRELEASED; urgency=medium
-
+cloud-init (23.3.2-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
```

Comment:
Just FYI, we increment the .1 at the end rather than 0ubuntu0 here. Doesn't need to change here though as this version will be overwritten with 23.3.3 before we upload.

Reply:
> we increment the .1 at the end rather than 0ubuntu0 here

Since this is adding a downstream patch, I think ubuntu1 makes more sense, right? The number after "ubuntu" is typically used to indicate patch revision on top of debian release. Since there is no debian release between us and upstream, (and this is literally adding a downstream patch) I would interpret this to mean patch revision on top of upstream release.

Reply:
SRU numbers are different. We should only increment the Y in `ubuntuY` during devel.
https://github.com/canonical/ubuntu-maintainers-handbook/blob/main/VersionStrings.md#version-adding-a-change-in-ubuntu-as-a-stable-release-update

Reply:
> SRU numbers are different

Ahhhh, I didn't realize our numbers at the end were an (SRU) exception to the normal package version schema. +1 Thanks, @TheRealFalcon.